src/templates/index.html

```diff
@@ -187,12 +187,22 @@
 
             function randomPadName()
             {
-                var chars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
-                var string_length = 10;
+                // the number of distinct chars (64) is chosen to ensure that
+                // the selection will be uniform when using the PRNG below
+                var chars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz-+";
```

Comment:
The `+` is a URL reserved character.
According to [RFC 3986](https://tools.ietf.org/html/rfc3986#section-2.3) the set of unreserved characters is:

```
unreserved  = ALPHA / DIGIT / "-" / "." / "_" / "~"
```

We should replace `+` with one of (`.`, `_`, `~`).
Should we go with `_`?